### PR TITLE
Adding docs for CKS cluster autoscaling

### PR DESCRIPTION
**Pull request by davidjumani**
Documentation for https://github.com/apache/cloudstack/pull/4329/files

### CODE REVIEW

**Comment by davidjumani** [comment]
requesting docbuild

**Comment by davidjumani** [comment]
requesting docbuild

File: source/plugins/cloudstack-kubernetes-service.rst [hunk 1]
```diff
@@ -272,6 +272,26 @@ Only running Kubernetes clusters can be scaled for size. When the service fails
...
+Auto-Scaling Kubernetes cluster
+###############################
+
+CloudStack has been officially added to the list of Kubernetes supported `Cloud Provides
+<https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#deployment>`_
+This allows Kubernetes to dynamically scale the size of the cluster based on the capacity requirements.
+Details on how the cluster-autoscaler works can be found on the official `Kubernetes cluster autoscaler repository
+<https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md>`_
+
+From 4.16, CloudStack supports automated deployment of the cluster autoscaler for Kubernetes Cluster versions 1.15.0 and higher.
```
Comment (rhtyd) [inline source/plugins/cloudstack-kubernetes-service.rst:284]: Should we say v1.16 if 1.15 is not supported?

**Review by rhtyd — APPROVED** [review]
LGTM, minor note added

**Comment by davidjumani** [comment]
Combining with #168

**Pull request closed without merging**